workflows/virtual-screening-blueprint/definition/main.nf

```diff
@@ -94,7 +89,7 @@ process RunMolMIMGenerate {
         val num_molecules
 
     output:
-        path "*.sdf", emit: sdfs
+        path "*.smi", emit: smiles
```

Comment:
confirming this is a bugfix?

Reply:
Yes, it looks like diffdock can take a list of SMILES strings, so we do not need to feed in SDFs any more, which used to causing problems.